.github/workflows/ci.feature.yml

```diff
@@ -105,6 +107,90 @@ jobs:
         run: |
           yarn workspace @superfluid-finance/ethereum-contracts test-coverage
 
+
+  #test automations:
+  test-automation-contracts-scheduler:
+    name: Build and test automation-contracts-scheduler (Feature Branch)
+
+    runs-on: ubuntu-latest
+
+    needs: [ check ]
+    if: needs.check.outputs.build_automation_contracts_scheduler
+
+    strategy:
+      matrix:
+        node-version: [ 16.x ]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Check changeset
```

Comment:
please delete this step, it is duplicated and unused in the job, I forgot to remove this from the ethereum-contracts job.